spec/models/link_spec.rb

```diff
@@ -155,4 +155,13 @@
     expect(@link.status).to be nil
     expect(@link.link_last_checked).to be nil
   end
+
+  describe "after_update" do
+    it "triggers a recount of the local authority's broken links" do
```

Comment:
Would it be worth adding another test here to demonstrate that `calculate_count_of_broken_links` will not get called if `status` does not change?